react/src/components/Map/Map.tsx

```diff
@@ -1,7 +1,86 @@
 import React from 'react';
+import { MapContainer, TileLayer, Marker, Popup } from 'react-leaflet';
+import MarkerClusterGroup from '@changey/react-leaflet-markercluster';
+import { TileServerLayer, FeatureCollection } from '../../types';
+import * as L from 'leaflet';
+import { LatLngTuple, MarkerCluster } from 'leaflet';
+import 'leaflet.markercluster';
 
-const Map = () => {
-  return <h2>Map</h2>;
+import 'leaflet/dist/leaflet.css';
+
+/* TODO: review if best approach is to style map with .leaflet-container */
+/* TODO: consider createTileLayerComponent
```

Comment:
`createTileLayerComponent` seems nice for situations when you have to update layer props!